apps/chrome/chrome.talon

```diff
@@ -1 +1,13 @@
+app: chrome
+-
+tag(): browser
 profile switch: user.chrome_mod("shift-m")
+
+tag(): user.tabs
```

Comment:
I don't think we need this or `tag(): browser`, that should be handled by eg. `chrome_linux.py`, `chrome_win.py`, etc.

Reply:
Thanks for your help!